var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -75,47 +83,28 @@ def install(self, spec, prefix):
             # completely disabled. See
             # [SuiteSparse/SuiteSparse_config/SuiteSparse_config.mk] for more.
             'CUDA=no',
-            'CUDA_PATH={0}'.format(
-                spec['cuda'].prefix if '+cuda' in spec else ''
-            ),
-            'CFOPENMP={0}'.format(self.compiler.openmp_flag)
-        ])
-
-        if '+pic' in spec:
-            make_args.extend([
-                'CFLAGS={0}'.format(self.compiler.pic_flag),
-                'FFLAGS={0}'.format(self.compiler.pic_flag)
-            ])
+            'CUDA_PATH=%s' % (spec['cuda'].prefix if '+cuda' in spec else ''),
+            'CFOPENMP=%s' % omp_flag,
+            'CFLAGS=-O3 -fexceptions -fno-common %s' % pic_flag,
```

Comment:
'-O3', '-fexceptions', and '-fno-common' are flags accepted by [GCC](https://gcc.gnu.org/onlinedocs/gcc/Option-Summary.html), [LLVM](https://clang.llvm.org/docs/CommandGuide/clang.html), and [Intel](https://software.intel.com/sites/default/files/m/d/4/1/d/8/icc.txt) compilers.

I don't see options with those semantics for PGI, except for `-O3`. I can get rid of `-fexceptions -fno-common`.